css/popup.css

```diff
@@ -0,0 +1,162 @@
+*,
+*::before,
+*::after {
+  -webkit-box-sizing: border-box;
+  -moz-box-sizing: border-box;
+  box-sizing: border-box;
+}
+
+a {
+  color: #FFF;
+  text-decoration: none;
+}
+
+ul {
+  list-style-type: none;
+  margin: 0px;
+  padding-left: 0px;
+}
+
+ul li span {
+  float: right;
+  margin-top: 9px;
+  font-size: 13px !important;
+  border: 2px solid rgba(255, 255, 255, 0.7);
+  display: inline-block;
+  text-align: center;
+  height: 30px;
+  font-weight: bold;
+  width: 30px;
+  line-height: 25px;
+  border-radius: 15px;
+}
+.close {
+  outline: none;
+  background-color: transparent;
+  border: 0px;
+  width: 20px;
+  height: 20px;
+  position: absolute;
+  top: 12px;
+  right: 15px;
+  cursor: pointer;
+  opacity: 0;
+  transition: all 0.4s cubic-bezier(0.4, 0, 0.2, 1) 0s;
+  transform: translateX(-60px) rotate(90deg);
+}
+.close.active {
+  transition: all 0.4s cubic-bezier(0.4, 0, 0.2, 1) 0.2s;
+  opacity: 1;
+  transform: translateX(0px) rotate(360deg);
+}
+.close .bar {
+  display: inline-block;
+  height: 2px;
+  width: 20px;
+  position: relative;
+  margin-bottom: 5px;
+  background-color: #FFF;
+}
+.close .bar:nth-of-type(1) {
+  transform: rotate(45deg) translateY(9px);
+}
+.close .bar:nth-of-type(2) {
+  transform: rotate(-45deg) translateY(-9px);
+}
+
+.share {
+  display: inline-block;
+  width: 180px;
+  height: 50px;
+  overflow: hidden;
+  line-height: 50px;
+  border-radius: 25px;
+  position: relative;
+  background-color:#D8D4D4;
+  transition: width 285ms cubic-bezier(0.4, 0, 0.2, 1) 0.2s, height 325ms cubic-bezier(0.4, 0, 0.2, 1) 0s;
+  text-align: center;
+  -webkit-box-shadow: 0px 5px 3px 0px rgba(0, 0, 0, 0.35);
+  -moz-box-shadow: 0px 5px 3px 0px rgba(0, 0, 0, 0.35);
+  box-shadow: 0px 5px 3px 0px rgba(0, 0, 0, 0.35);
+}
+.share .toggle {
+  display: block;
+  width: 100%;
+}
+.share.active {
+  width: 350px;
+  height: 230px;
+  transition: width 285ms cubic-bezier(0.4, 0, 0.2, 1) 0.1s, height 325ms cubic-bezier(0.4, 0, 0.2, 1) 0.2s;
+  -webkit-box-shadow: 0px 5px 3px 4px rgba(0, 0, 0, 0.25);
+  -moz-box-shadow: 0px 5px 3px 4px rgba(0, 0, 0, 0.25);
+  box-shadow: 0px 5px 3px 4px rgba(0, 0, 0, 0.25);
+}
+.share.active .content {
+  opacity: 1;
+  transform-origin: center center;
+  height: 170px;
+  transform: scale(1, 1);
+  transition: all 0.4s cubic-bezier(0.4, 0, 0.2, 1) 0.3s;
+}
+.share.active .content .text {
+  transition-delay: 0.6s;
+  opacity: 1;
+  transform: translateY(0px);
+}
+.share.active .toggle span {
+  transition: all 0.4s cubic-bezier(0.4, 0, 0.2, 1) 0.2s;
+  transform: translateX(-128px);
+}
+.share span {
+  text-transform: uppercase;
+  font-size: 18px;
+  color: orange;
+  letter-spacing: 1px;
+  display: inline-block;
+  transition: all 0.4s cubic-bezier(0.4, 0, 0.2, 1) 0s;
+}
+
+
+.share .content {
+  transition: all 0.4s cubic-bezier(0.4, 0, 0.2, 1) 0s;
+  width: 100%;
+  height: 0px;
+  padding: 10px 30px;
+  text-align: left;
+  position: absolute;
```

Comment:
no use of: 
 position: absolute 
 top: 0
 left: 0
It works just fine without defining them